providers/summary/scancode.js

```diff
@@ -153,9 +154,16 @@ class ScanCodeSummarizer {
     return result.length === 0 ? null : result
   }
 
-  _collectAttributions(copyrights, holders) {
+  _collectAttributions(copyrights) {
     if (!copyrights || !copyrights.length) return null
-    return this._setToArray(copyrights.reduce((result, entry) => this._addArrayToSet(entry.holders, result), new Set()))
+    const set = new Set()
+    for (let copyright of copyrights) {
+      if (!copyright.statements) continue
+      const statements = copyright.statements.map(statement => he.decode(statement)
+        .replace(/\\[nr]/, '\n'))
+      this._addArrayToSet(statements, set)
+    }
+    return this._setToArray(set)
```

Comment:
Good for now. Going forward there are some additional cleanups we can do like normalizing punctuation at the end of the line, removing duplicate white spaces, trimming both ends of the string, ...

Reply:
Acknowledged.